pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh

```diff
@@ -71,6 +71,27 @@ elif command -v dnf >/dev/null 2>&1; then
 		# Workaround for https://github.com/containerd/stargz-snapshotter/issues/340
 		ln -s fusermount3 /usr/bin/fusermount
 	fi
+elif command -v yum >/dev/null 2>&1; then
+	pkgs=""
+	yum_install_flags="-y"
+	if ! rpm -ql epel-release >/dev/null 2>&1; then
+		yum install ${yum_install_flags} epel-release
+	fi
+	if ! command -v dnf >/dev/null 2>&1; then
+		pkgs="${pkgs} dnf"
+	fi
+	if [ "${LIMA_CIDATA_MOUNTS}" -gt 0 ] && ! command -v sshfs >/dev/null 2>&1; then
+		pkgs="${pkgs} fuse-sshfs"
+	fi
+	if [ "${LIMA_CIDATA_CONTAINERD_USER}" = 1 ]; then
+		if ! command -v mount.fuse3 >/dev/null 2>&1; then
+			pkgs="${pkgs} fuse3"
+		fi
+	fi
+	if [ -n "${pkgs}" ]; then
+		# shellcheck disable=SC2086
+		yum install ${yum_install_flags} ${pkgs}
+	fi
```

Comment:
Why not install other packages such as shadow-utils and iptables?

Reply:
This packages was already there by default, that's why I didn't feel the need to add them. Do you think it's relevant anyway? The only situation it's a problem is if someone decides to remove the package (no matter why).

Reply:
I'd prefer to add them for consistency with EL8 family.
Also, some variant of EL7 may not have them installed by default.

Reply:
Alright. Added.